### PR TITLE
Fix multiline align imports

### PR DESCRIPTION
**Pull request by skluyskens-azumuta**
fixes DEV-1377

Warning: with these changes, the imported functions etc are sorted, we might also try to make the imports more compact, ...

To test:
npm un eslint-plugin-azumuta
npm i ../eslint-plugin-azumuta

Should apply this renaming in one commit to avoid pull requests with formatting changes

### CODE REVIEW

File: lib/rules/align-imports.js [hunk 1]
```diff
@@ -22,15 +23,17 @@ function splitSpecifiers(line, expectedFromPosition) {
...
-    });
+    // reset global regex
+    importWithCurlyBracketsPart.lastIndex = 0;
+    const resultPart1 = importWithCurlyBracketsPart.exec(line);
+    const elements = resultPart1?.groups?.elements;
+    const file = resultPart1?.groups?.file;
+    if (elements && file) {
+        const parts = elements.split(/[,\s]+/g).filter(p => p);
+        // TODO check if we want to sort
+        //parts.sort();
```
Comment (skluyskens-azumuta) [inline lib/rules/align-imports.js:34]: Do we want to sort the imported objects?

**Review by SimonVanneste — COMMENTED** [review]
Are there any tests for when single line imports are very close (smaller, equal and bigger) to the max for one line. There were some bugs there in the past.

File: lib/alignImports.test.js [hunk 2]
```diff
@@ -4,6 +4,109 @@ const rule = require('./rules/align-imports');
 
 ruleTester.run('AlignImports', rule, {
     valid: [
+        // don't modify what can stay on one line
+        {
+            code: `
+import Balances, { Balance }            from '/imports/api/balances';`,
+        },
+        // don't modify what can't stay on one line (corner case)
```
Comment (skluyskens-azumuta) [inline lib/alignImports.test.js:30]: Exactly one character too much to be able to fit on one line

File: lib/alignImports.test.js [hunk 3]
```diff
@@ -4,6 +4,109 @@ const rule = require('./rules/align-imports');
...
+            code: `
+import JumpToTarget, {
+    Rule,
+    RuleType2,
+}                                       from 'common/api/rules';`,
+        },
+        // don't modify what can stay on one line (corner case)
+        {
+            code: `
+import { JumpToTarget, Rule, RuleType } from 'common/api/rules';`,
```
Comment (skluyskens-azumuta) [inline lib/alignImports.test.js:50]: Exact number of characters to fit on one line

**Comment by skluyskens-azumuta** [comment]
> Are there any tests for when single line imports are very close (smaller, equal and bigger) to the max for one line. There were some bugs there in the past.

Yes, quite some tests were added

**Review by SimonVanneste — APPROVED** [review]
